File: 27-JS/seeds/seeds.js
var square_number = 0
var total_seeds = 0
// Loops through for each square
for(var i = 1; i <= 64; i++) {
	var square_seed = Math.pow(2, (i-1) );
	console.log("Square " + i + ": " + square_seed);
	square_number += 1;
	total_seeds += square_seed;
}
// Prints out total number of seeds
console.log("The total number of seeds are " + total_seeds);
// Function so you can call for the seed number when you want
var seed_count = function(square_number) {
	var square_seed = Math.pow(2, (square_number-1));
	console.log("There are " + square_seed + " seeds on the square number " + square_number);
}

File: 27-JS/seeds/seeds.rb
require 'pry'
total_seeds = 0
for i in 1..64 
	seed_number = 2**(i-1)
	total_seeds += seed_number
	puts "There are #{seed_number} seeds on square #{i}"
end

def seed_count(square_number) 
	number_of_seeds = 2**(square_number-1)
	puts "There are #{number_of_seeds} seeds on square #{square_number}."
end

puts "There are #{total_seeds} on the total chess board."

binding.pry